scripts/ubuntu-bionic/00-setup.sh

```diff
@@ -18,8 +18,7 @@ fail() {
     exit 1
 }
 
-# TODO: NEED_KERNEL_SOURCE? REBUILD_MODULES?
-REBUILD_KERNEL=false
+OLD_KERNEL=false
```

Comment:
What does this mean in this case?  I should probably remember, but I don't :)

Reply:
https://github.com/taskcluster/taskcluster/issues/3574 helped me remember: this is because
> -aws kernels do not include snd-aloop at all (not even as a module) 

so this basically entails continuing to use the hand-built kernel.  Let's add a comment.